charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.109.0
+version: 0.109.1
```

Comment:
run `make generate-examples CHARTS=opentelemetry-collector` to generate the examples.